crates/polars-lazy/src/scan/ipc.rs

```diff
@@ -44,9 +44,8 @@ impl LazyIpcReader {
 }
 
 impl LazyFileListReader for LazyIpcReader {
-    fn finish_no_glob(self) -> PolarsResult<LazyFrame> {
+    fn finish_no_glob(self, path: &Path) -> PolarsResult<LazyFrame> {
```

Comment:
This and similar changes should actually be `load_specific(self, reader: ReaderFactory)`.